guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -115,13 +115,6 @@
     background-image: url("images/guac-tricolor.png");
 }
 
-.login-ui.continuation div.login-dialog {
-    border-right: none;
-    border-left: none;
-    box-shadow: none;
-    max-width: 6in;
-}
-
```

Comment:
The main other piece of code which leverages the authentication continuation process is the database password reset. Does the password reset still look OK with these changes?